src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Pkcs12.cs

```diff
@@ -10,5 +12,33 @@ internal static partial class Crypto
     {
         [DllImport(Libraries.CryptoNative)]
         internal static unsafe extern SafePkcs12Handle DecodePkcs12(byte[] buf, int len);
+
+        [DllImport(Libraries.CryptoNative)]
+        internal static extern SafePkcs12Handle D2IPkcs12Bio(SafeBioHandle bio);
```

Comment:
Really I liked that we had gotten rid of "D2I" (by changing it to "Decode"), so something better than bringing D2I back is what I'm going for.